presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -172,12 +172,28 @@ class SystemConfig : public ConfigBase {
 
   int httpServerHttpsPort() const;
 
+  // This config controls what cipher suites are supported by Native workers for
```

Comment:
> // This config controls what cipher suites a

Consider to shorten to "A list of supported ciphers."

The return type is a std::string. Does it contain the list of ciphers separated by comma or something? Would be nice to clarify.

Reply:
Yes. It's a comma separated list like `AES128-SHA,AES128-SHA256,AES256-GCM-SHA384`